packages/core/src/components/Snackbar/SnackbarProvider/SnackbarProvider.tsx

```diff
@@ -27,7 +42,7 @@ export interface HvNotistackSnackMessageProps {
   /** Classname to apply on the root node */
   className?: string;
   /** Your component tree. */
-  message?: string;
+  message?: ReactNode | string;
```

Comment:
this is redundant because `ReactNode` already includes `string`:

```tsx
type ReactNode = string | number | boolean | ReactElement<any, string | JSXElementConstructor<any>> | Iterable<ReactNode> | ReactPortal | null | undefined
```